core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -415,6 +418,7 @@ public boolean hasBlockMeta(long blockId) {
 
   @Override
   public long lockBlock(long sessionId, long blockId) {
+    Metrics.WORKER_ACTIVE_CLIENTS.inc();
     return mLocalBlockStore.lockBlockNoException(sessionId, blockId);
```

Comment:
should we still count the client as an active client if the lock acquisition failed? (e.g. returned invalid block id?)